### PR TITLE
Updated info about installation to relevant

### PR DESCRIPTION
**Pull request by idchlife**
Because at first I really added folder and file to iCloud Documents. Then I thought - maybe it's Python Modules?

### CODE REVIEW

**Comment by zrzka** [comment]
screen shot 2018-05-20 at 22 06 40

I'd happily merge change like this, but can you rephrase it? I like your change, but the wording / order is kinda weird.

**Comment by idchlife** [comment]
np, will try to fix it

**Comment by idchlife** [comment]
@zrzka tried to fix it.

**Comment by idchlife** [comment]
@zrzka does this PR need another edit or is it good now? Thanks!

**Comment by zrzka** [comment]
@idchlife holidays this week, patience please. Hard to review, merge and release on phone only.

**Comment by idchlife** [comment]
@zrzka no problem! Not rushing or anything, just pinging (sorry if it bothers or irritates, just had experience with repos where it's ok to ping since well we all busy people and sometimes forget stuff etc)

**Comment by zrzka** [comment]
@idchlife don't worry, feel free to ping me anytime. It doesn't bother me at all!
